### PR TITLE
[rails-13] Adds teams controller

### PR DESCRIPTION
**Pull request by cpiazza**

### CODE REVIEW

File: app/assets/javascripts/teams.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (cpiazza) [inline app/assets/javascripts/teams.coffee]: we can delete this file

File: app/controllers/teams_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class TeamsController < ApplicationController
```
Comment (cpiazza) [inline app/controllers/teams_controller.rb]: this file is in the wrong directory. it should be under `app/controllers/api/v1`. also, the class name should be: `Api::V1::TeamsController < ApplicationController`

File: app/views/api/v1/teams/index.json.builder [hunk 3]
```diff
@@ -0,0 +1 @@
+json.teams @teams, :id, :name, :color
```
Comment (cpiazza) [inline app/views/api/v1/teams/index.json.builder]: this file is named wrong - the extension should be `.jbuilder`

File: config/routes.rb [hunk 4]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get 'teams/index'
```
Comment (cpiazza) [inline config/routes.rb]: route is wrong. should be below like the `:users` route

File: app/assets/javascripts/teams.coffee (hunk 1 above)
Comment (alpiazza13) [reply app/assets/javascripts/teams.coffee]: check

File: app/controllers/teams_controller.rb (hunk 2 above)
Comment (alpiazza13) [reply app/controllers/teams_controller.rb]: check

File: app/views/api/v1/teams/index.json.builder (hunk 3 above)
Comment (alpiazza13) [reply app/views/api/v1/teams/index.json.builder]: check

**Pull request merged**